src/ui/public/registry/doc_table_detail_views.js

```diff
@@ -0,0 +1,7 @@
+define(function (require) {
+  return require('ui/registry/_registry')({
+    name: 'docTableDetailViews',
+    index: ['title'],
+    order: ['order']
```

Comment:
I went with the constructor approach.  I also added a new field called "name".  It defaults to title but is meant for the case where title has quotes or other strange characters that might mess up the docViews[title] code.  So, now you can specify a name in that case.